src/SelectField/SelectField.jsx

```diff
@@ -0,0 +1,266 @@
+import ThemeManager from '../styles/theme-manager';
+import ContextPure from '../mixins/context-pure';
+import deprecated from '../utils/deprecatedPropType';
+
+const SelectField = React.createClass({
+
+  mixins: [
+    StylePropable,
+    ContextPure,
+  ],
+
+  contextTypes: {
+    muiTheme: React.PropTypes.object,
+  },
+
+  statics: {
+    getChildrenClasses() {
+      return [
+        TextField,
+        DropDownMenu,
+      ];
+    },
+  },
+
+  propTypes: {
+    /**
+     * The width will automatically be set according to the
+     * items inside the menu. To control this width in css
+     * instead, set this prop to `false`.
+     */
+    autoWidth: React.PropTypes.bool,
+
+    /**
+     * The `MenuItem` elements to populate the `Menu` with.
+     * If the MenuItems have the prop `label` that value will
+     * be used to render the representation of that
+     * item within the field.
+     */
+    children: React.PropTypes.node,
+
+    /**
+     * Disables the select field if set to true.
+     */
+    disabled: React.PropTypes.bool,
+
+    /**
+     * The style object to use to override error styles.
+     */
+    errorStyle: React.PropTypes.object,
+
+    /**
+     * The error content to display.
+     */
+    errorText: React.PropTypes.node,
+
+    /**
+     * The style object to use to override floating label styles.
+     */
+    floatingLabelStyle: React.PropTypes.object,
+
+    /**
+     * The content to use for the floating label element.
+     */
+    floatingLabelText: React.PropTypes.node,
+
+    /**
+     * If true, the field receives the property width 100%.
+     */
+    fullWidth: React.PropTypes.bool,
+
+    /**
+     * The hint content to display.
+     */
+    hintText: React.PropTypes.node,
+
+    /**
+     * Overrides the styles of the icon element.
+     */
+    iconStyle: React.PropTypes.object,
+
+    /**
+     * `SelectField` will use text as default value,
+     * with this property you can choose another name.
+     */
+    labelMember: deprecated(React.PropTypes.string,
+      'to promote composability.'),
+
+    /**
+     * Overrides the styles of label when the `SelectField` is inactive.
+     */
+    labelStyle: deprecated(React.PropTypes.object,
+      'to promote composability.'),
+
+    /**
+     * JSON data representing all menu items in the dropdown.
+     */
+    menuItems: deprecated(React.PropTypes.array,
+      'to promote composability.'),
+
+    /**
+     * Callback function that is fired when the `SelectField` loses focus.
+     */
+    onBlur: React.PropTypes.func,
+
+    /**
+     * Callback function that is fired when the value changes.
+     */
+    onChange: React.PropTypes.func,
+
+    /**
+     * Callback function that is fired when the `SelectField` gains focus.
+     */
+    onFocus: React.PropTypes.func,
+
+    /**
+     * The style object to use to override the `DropDownMenu`.
+     */
+    selectFieldRoot: React.PropTypes.object,
```

Comment:
definitely, I was like HOLY JESUS WHAT IS THIS!!! 

we shouldn't introduce a lot of breaking changes. This has to wait a little longer -_-